unit_test/blas/Test_Blas1_abs.hpp

```diff
@@ -153,7 +153,7 @@ int test_abs() {
   Test::impl_test_abs<view_type_a_ll, view_type_b_ll, Device>(0);
   Test::impl_test_abs<view_type_a_ll, view_type_b_ll, Device>(13);
   Test::impl_test_abs<view_type_a_ll, view_type_b_ll, Device>(1024);
-  Test::impl_test_abs<view_type_a_ll, view_type_b_ll, Device>(132231);
+  //Test::impl_test_abs<view_type_a_ll, view_type_b_ll, Device>(132231);
```

Comment:
I thought this was already pushed as part of another commit.

Reply:
That is on Trilinos. This includes the PR for kokkoskernels.

Reply:
Got it, Thanks !